### PR TITLE
Use generated types to construct partial types

### PR DESCRIPTION
**Pull request by ishrak-imam**
We were constructing the partial types without taking into account the codegen generated types.
Because of that if we change the schema for those partial types and do codegen there will be no TS errors in the resolvers which can force us to fix/adjust them.

In this PR all the partial types are modified to construct them using the generated types.

### CODE REVIEW

File: subschemas/substrate-chain/src/resolvers/Query/auctions.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
-
 import {ApiPromise} from '@polkadot/api';
 import {createWsEndpoints} from '@polkadot/apps-config/endpoints';
 import type {LinkOption} from '@polkadot/apps-config/endpoints/types';
-import type {ITuple, Registry, Codec} from '@polkadot/types/types';
```
Comment (ishrak-imam) [inline subschemas/substrate-chain/src/resolvers/Query/auctions.ts:5]: This was just an unused leftover. Other changes in this file are only prettier formatting :)

File: subschemas/substrate-chain/src/services/crowdloanService.ts [hunk 2]
```diff
@@ -31,8 +31,8 @@ interface Campaigns {
 
 export async function getFunds(paraIds: ParaId[], bestNumber: BlockNumber, api: ApiPromise): Promise<Campaigns> {
   const [rawFunds, rawLeases] = await Promise.all([
-    api.query.crowdloan?.funds?.multi<Option<FundInfo>>(paraIds),
-    api.query.slots?.leases?.multi(paraIds) as unknown as Option<ITuple<[AccountId, BalanceOf]>>[][],
+    api.query.crowdloan.funds.multi<Option<FundInfo>>(paraIds),
```
Comment (ishrak-imam) [inline subschemas/substrate-chain/src/services/crowdloanService.ts:34]: @franjs Do we need the optional chaining here?

**Review by jdomingos — APPROVED** [review]
Nice, thanks @ishrak-imam! 👍

**Review by franjs — APPROVED** [review]
Awesome! 👍🏼

**Pull request merged**